packages/cashier/src/Components/Error/__tests__/form-error.spec.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import FormError from '../form-error';
+import { createBrowserHistory } from 'history';
+import { Router } from 'react-router';
+import { cleanup, fireEvent, render, screen, waitFor } from '@testing-library/react';
+import { routes } from '@deriv/shared';
+
+jest.mock('Stores/connect.js', () => ({
+    __esModule: true,
+    default: 'mockedDefaultExport',
+    connect: () => Component => Component,
+}));
+
+describe('<FormError />', () => {
+    beforeAll(() => {
+        const modal_root_el = document.createElement('div');
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(modal_root_el);
+    });
+
+    it('should show "Please verify your identity" message, "Cancel" and "Verify identity" buttons', () => {
+        render(<FormError error={{ code: 'Fiat2CryptoTransferOverLimit', message: 'Error is occured' }} />);
+
+        expect(screen.getByText('Please verify your identity')).toBeInTheDocument();
+        expect(screen.getByText('Cancel')).toBeInTheDocument();
+        expect(screen.getByText('Verify identity')).toBeInTheDocument();
+    });
+
+    it('should redirect to "/account/proof-of-identity" page, when "Verify identity" button was clicked', () => {
+        const history = createBrowserHistory();
+        const setErrorMessage = jest.fn();
+        const error = {
+            code: 'Fiat2CryptoTransferOverLimit',
+            message: 'Error is occured',
+            setErrorMessage,
+        };
+        const expected_route = 'proof_of_identity';
+        render(
+            <Router history={history}>
+                <FormError error={error} />
+            </Router>
+        );
+        const on_confirm_btn = screen.getByText('Verify identity');
+        fireEvent.click(on_confirm_btn);
+
+        expect(history.location.pathname).toBe(routes[expected_route]);
```

Comment:
can we use `routes.proof_of_identity` here instead of declaring a new variable?

Reply:
ok sure